aggregator/src/aggregator/http_handlers.rs

```diff
@@ -266,6 +267,32 @@ where
     }
 }
 
+/// A Trillium handler that returns an empty body with retry-after and location headers.
+#[derive(Clone)]
+struct EmptyBody {
+    location: String,
+}
+
+impl EmptyBody {
+    /// Return an EmptyBody with the location set to the relative path to the
+    /// aggregation job for the task.
+    fn for_task_and_job(task_id: &TaskId, aggregation_job_id: &AggregationJobId) -> Self {
+        Self {
+            location: format!("/tasks/{task_id}/aggregation_jobs/{aggregation_job_id}?step=0"),
+        }
+    }
+}
+
+#[async_trait]
+impl Handler for EmptyBody {
+    async fn run(&self, conn: Conn) -> Conn {
+        conn.with_response_header(KnownHeaderName::RetryAfter, "2")
```

Comment:
I think that hard coding 2 seconds is fine if our goal right now is to get the message format changes in. But let's log an issue about computing a better value here, based on aggregation job driver poll interval?

Reply:
```suggestion
        // To be fixed in issue #3921
        conn.with_response_header(KnownHeaderName::RetryAfter, "2")
```